pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -11525,6 +11596,14 @@
       call_to_action_learn_more: "Learn more"
       aria_label_learn_more: "Learn more about App Lab"
       call_to_action_try: "Try App Lab"
+    musiclab:
+      overline: "Ages 10+"
+      title: "Music Lab"
+      what_you_can_make: "Create your own mix and beats with code!"
+      experience: "Beginner"
+      call_to_action_learn_more: "Learn more"
+      aria_label_learn_more: "Learn more about Music Lab"
+      call_to_action_try: "Try Music Lab"
```

Comment:
Let's move this under the `tools_page` namespace above. This makes the translation process easier in CrowdIn (the tool we use to connect with translators).